tests/local/warehouse/validity/test_date_and_time_validity_formats.py

```diff
@@ -21,7 +22,7 @@ class TestDateAndTimeValidityFormats(SqlTestCase):
     def test_date_eu(self):
         self.sql_recreate_table(
             [f"name {self.dialect.data_type_varchar_255}"],
-            ["('21-01-2021')",
```

Comment:
Any reason why this is removed? I think we support date validation with `-` in it